__tests__/commands/install.js

```diff
@@ -775,3 +775,25 @@ test.concurrent('install uses OS line endings when lockfile doesn\'t exist', asy
       assert(lockfile.indexOf(os.EOL) >= 0);
     });
 });
+
+describe('authed registries', (): void => {
```

Comment:
Any reason not to split this test into individual `describe`?
I think it should work fine along with the others.